src/EditorFeatures/Test/Workspaces/TestWorkspaceFactory_XmlConsumption.cs

```diff
@@ -225,6 +225,11 @@ public static TestWorkspace CreateWorkspace(string xmlDefinition, bool completed
             if (projectElement.Attribute(FilePathAttributeName) != null)
             {
                 filePath = projectElement.Attribute(FilePathAttributeName).Value;
+                if (string.Compare(filePath, "null", StringComparison.Ordinal) == 0 || string.Compare(filePath, "Nothing", StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
Option B: allow empty string to mean null (unless we allow that distinguishing in the workspaces API...), or have a FilePathNull="true" attribute to make it really explicit.